BaseApp/COD66601.TXT

```diff
@@ -0,0 +1,182 @@
+OBJECT Codeunit 66601 RegexOptions

```

Comment:
Please follow our coding guidelines with regards to naming etc. Those guidelines were very well hidden, for which I apologize. I've now moved them to the frontpage.